### DISCUSSION TITLE
Clarify Policy priority requirements

### DISCUSSION THREAD

**Discussion by zanebclark** [openmobilityfoundation/curb-data-specification#84] [Data Specification]
The requirements around [Policy](https://github.com/openmobilityfoundation/curb-data-specification/tree/main/curbs#policy) `priority` came up during the 2/22 meeting.
>Two Policies that apply to the same Curb Zone with overlapping Time Spans and user classes MUST NOT have the same priority.

@schnuerle and @jacobmalleau suggested that two policies w/ overlapping `time_spans` and overlapping `user_classes` _could_ have the same priority so long as the `user_classes` arrays don't consist of the same values. (I was going to say, "so long as the `user_classes` arrays aren't equivalent". See https://github.com/openmobilityfoundation/curb-data-specification/discussions/82 for more on this). If this is the intent of the spec, we could reword the above requirement: 
>Two Policies that apply to the same Curb Zone with overlapping Time Spans and **_equivalent_** User Class enumerations MUST NOT have the same priority.

**Comment by jacobmalleau** [comment]
I agree with this.

**Comment by schnuerle** [comment]
I've updated the priority definition with your clarified language, thanks!

https://github.com/openmobilityfoundation/curb-data-specification/blob/release-1.0.0/curbs/README.md#policy
